manifests/t/tanshuai/alphabiz/0.1.0.0/tanshuai.alphabiz.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: tanshuai.alphabiz
+PackageVersion: 0.1.0.0
+PackageLocale: en-US
+Publisher: Zeeis

```

```suggestion
Publisher: Alphabiz Team
```
This must match Control Panel.